index.js

```diff
@@ -257,10 +272,25 @@ Hermes.prototype.unsubscribe = function (queueName, handler, cb) {
 };
 
 /**
+ * Disconnect
+ * @param {Function} cb
+ * @return this
+ */
+Hermes.prototype.close = function (cb) {
+  debug('hermes close');
+  this._channel.close(function () {
+    debug('hermes close success', arguments);
+    cb.apply(cb, arguments);
```

Comment:
because first argument would be `this` for the `cb`. So inside that callback you would be able to call `this()`?
I've never seen that. Usually people pass `null`.